### PR TITLE
Ref [TASK][38700] Blog details page - navbar completed.

### PR DESCRIPTION
**Pull request by hi-long**
- Link redmine: https://edu-redmine.sun-asterisk.vn/issues/38700

### CODE REVIEW

**Comment by hi-long** [comment]
ready

File: src/pug/pages/blog-details.pug [hunk 1]
```diff
@@ -1,4 +1,48 @@
...
+            //- BLOG NAV
+            .col-3.d-none.d-lg-block.blog-details__nav 
+                //- NAV NEWEST BLOGS
+                h2.fs-4.fw-normal BLOG
+                h3.fs-6.fw-normal BÀI VIẾT MỚI NHẤT
+                hr
+                .blog-details__nav-newest-blogs 
+                    +blog-recommend(blogs[0])
+                    +blog-recommend(blogs[1])
+                    +blog-recommend(blogs[2])
```
Comment (vietnq-1097) [inline src/pug/pages/blog-details.pug]: for loop

File: src/scss/pages/_blog-details.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.blog-details {
+    padding-top: 6rem;
+    &__main,
```
Comment (vietnq-1097) [inline src/scss/pages/_blog-details.scss]:
```suggestion
    padding-top: 6rem;
    
    &__main,
```

File: src/scss/pages/_blog-details.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        @include mediaQueryMin('large') { margin-top: 4.5rem; }
+    }
+
+    &__tags {
+        overflow-x: auto;
+        .tag {
+            color: black;
+        }
+    }
+}
```
Comment (vietnq-1097) [inline src/scss/pages/_blog-details.scss:23]: Thêm 1 dòng endline nhé

**Comment by hi-long** [comment]
ready

**Pull request merged**